src/components/TerminalApp.tsx

```
@@ -419,16 +455,16 @@ function TerminalInner({ initialCommand, active = true, onTabAction }: TerminalA
         if (ev.code !== 1000) {
           term.writeln(`\r\n\x1b[33m[Disconnected — will retry in 3s…]\x1b[0m`);
           reconnectTimerRef.current = setTimeout(() => {
-            if (mountedRef.current) connect();
+            if (mountedRef.current) connect().catch(releaseAfterFailedConnect);
```

ESLint

[error] Error: Cannot access variable before it is declared

`connect` is accessed before it is declared, which prevents the earlier access from updating when this value changes over time.

/home/jailuser/git/src/components/TerminalApp.tsx:458:37
  456 |           term.writeln(`\r\n\x1b[33m[Disconnected — will retry in 3s…]\x1b[0m`);
  457 |           reconnectTimerRef.current = setTimeout(() => {
> 458 |             if (mountedRef.current) connect().catch(releaseAfterFailedConnect);
      |                                     ^^^^^^^ `connect` accessed before it is declared
  459 |           }, 3000);
  460 |         }
  461 |       }

/home/jailuser/git/src/components/TerminalApp.tsx:231:3
  229 |   }, [updateStatus]);
  230 |
> 231 |   const connect = useCallback(async () => {
      |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 232 |     if (!mountedRef.current || !containerRef.current) return;
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 233 |     if (connectLockRef.current) return;
      …
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 462 |     };
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 463 |   }, [wsUrl, updateStatus, releaseAfterFailedConnect]);
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `connect` is declared here
  464 |
  465 |   useEffect(() => {
  466 |     mountedRef.current = true;

(react-hooks/immutability)